<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminForm</class>
 <widget class="QWidget" name="adminForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="viewsAdmin">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>631</width>
     <height>471</height>
    </rect>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>221</width>
       <height>331</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>INSERT MATERIAS</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QLineEdit" name="sNameInsertLE">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="sNrcInsertLE">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="sCodeInsertLE">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>90</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="sScheduleInsertLE">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>120</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="subjectNameLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre:</string>
      </property>
     </widget>
     <widget class="QLabel" name="subjectNrcLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>47</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>NRC:</string>
      </property>
     </widget>
     <widget class="QLabel" name="subjectCodeLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>47</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Clave:</string>
      </property>
     </widget>
     <widget class="QLabel" name="subjectScheduleLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Horario:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="sDayInsertLE">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>150</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="sSectionInsertLE">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>210</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="subjectInsertPB">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>280</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Ejecutar Insert</string>
      </property>
     </widget>
     <widget class="QLabel" name="sDayLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>47</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Día:</string>
      </property>
     </widget>
     <widget class="QLabel" name="sSectionLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Sección:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="sCodeSubjectInsertLE">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>240</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="sCodeSubjectLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Código Materia:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="sClassroomInsertLE">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>180</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="subjectClassroomLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>47</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Salón:</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>180</y>
       <width>251</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>DELETE MATERIAS</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QLineEdit" name="subjectNrcDeleteLE">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="subjectNrcDeleteLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>NRC:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="subjectDeletePB">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>70</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Ejecutar Delete</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="modifyTableTeacherPB">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>310</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Modificar Maestros</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>10</y>
       <width>251</width>
       <height>151</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>UPDATE MATERIAS</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QLineEdit" name="subjectSetDataLE">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>30</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="subjectWhereDataLE">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>60</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="subjectSetLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Nuevo Salón:</string>
      </property>
     </widget>
     <widget class="QLabel" name="subjectWhereLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>NRC:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="subjectUpdatePB">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>102</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Ejecutar Update</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="subjectDisplayLogPB">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>310</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Mostrar Logs</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>231</width>
       <height>211</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>INSERT MAESTROS</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QLineEdit" name="teacherIDLE">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>30</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="teacherNameLE">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>60</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="teacherLastNameLE">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>90</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="teacherCodeSubjectLE">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>120</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="teacherIDLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>ID:</string>
      </property>
     </widget>
     <widget class="QLabel" name="teacherNameLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>47</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre:</string>
      </property>
     </widget>
     <widget class="QLabel" name="teacherLastNameLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Apellidos:</string>
      </property>
     </widget>
     <widget class="QLabel" name="teacherCodeSubjectLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Código Materia:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="teacherInsertPB">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>160</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Ejecutar Insert</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>10</y>
       <width>251</width>
       <height>151</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>UPDATE MAESTROS</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QLineEdit" name="teacherCodeSubjectUpdateLE">
      <property name="geometry">
       <rect>
        <x>132</x>
        <y>30</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="teacherIdUpdateLE">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>60</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="teacherCodeSubjectUpdateLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Código Materia:</string>
      </property>
     </widget>
     <widget class="QLabel" name="teacherIdUpdateLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>ID:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="teacherUpdatePB">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>100</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Ejecutar Update</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>180</y>
       <width>251</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>DELETE MAESTROS</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QLineEdit" name="teacherIdDeleteLE">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="teacherIdDeleteLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>ID:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="teacherDeletePB">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>70</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Ejecutar Delete</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="modifyTableSubjectPB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Modificar Materias</string>
     </property>
    </widget>
    <widget class="QPushButton" name="teacherDisplayLogPB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Mostrar Logs</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QTableView" name="logTableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>591</width>
       <height>181</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="displayTeacherLogPB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Logs Actualizaciones Maestros</string>
     </property>
    </widget>
    <widget class="QPushButton" name="displaySubjectLogPB">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>230</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Logs Actualizaciones Materias</string>
     </property>
    </widget>
    <widget class="QPushButton" name="modifySubjectLogPB">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>280</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Modificar Materias</string>
     </property>
    </widget>
    <widget class="QPushButton" name="modifyTeacherLogPB">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>280</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Modificar Maestros</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteSubjectLogPB">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>230</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Logs Eliminaciones Materias</string>
     </property>
    </widget>
    <widget class="QPushButton" name="resetLogsPB">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>280</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Reiniciar Tabla</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deletLogTeacherPB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Logs Eliminaciones Maestros</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
